pkg/cmd/codespace/delete.go

```diff
@@ -187,6 +190,17 @@ func (a *App) Delete(ctx context.Context, opts deleteOptions) (err error) {
 		}
 		return nil
 	})
+
+	if a.io.IsStdoutTTY() && deletedCodespaces > 0 {
+		cs := a.io.ColorScheme()
+		successMsg := fmt.Sprintf("%s Successfully deleted %d codespaces\n", cs.SuccessIcon(), deletedCodespaces)
+		if len(codespacesToDelete) == 1 {
+			successMsg = fmt.Sprintf("%s Successfully deleted %s\n", cs.SuccessIcon(), codespacesToDelete[0].Name)
+		}
+		fmt.Fprint(a.io.Out, successMsg)
```

Comment:
In the past when we've added new text to stdout/stderr we've gotten complaints that we've broken (or just dirtied up) customer scripts. Do you think we should add a `--quiet` type flag to disable this message?

Reply:
@cmbrose Throughout gh commands we only print confirmation messages 1) to stderr and 2) only if stdout is connected to a terminal. With that approach we signal to the user that this output is for human readability only and won't be present for (nor affect) scripts in any way.

Reply:
Updated to use `ErrOut`